deploy/prometheus-k8s-role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
According to our operator standards, this file should be in a directory called `manifests`. (I know this operator was made back before we had that standard, but if it's easy enough to `git mv` it, we may as well).

Reply:
We have a JIRA card to track fixing this, so I think it will be done in a future PR.